StatusBar.c

```
@@ -0,0 +1,190 @@
+      }
+
+      token = end;
+   }
+}
+
+#if defined(HTOP_LINUX) && defined(HAVE_SENSORS_SENSORS_H)
+static void StatusBar_appendSensorValue(char* buffer, size_t size, HardwareSensorType type, const char* label, double value, char temperatureUnit) {
+   size_t used = strlen(buffer);
+   if (used >= size - 1)
+      return;
+
```

Cppcheck (2.21.0)

[style] The function 'HeaderLayout_fromName' is never used.

(unusedFunction)